task/git-cli/0.4/git-cli.yaml

```diff
@@ -125,8 +133,9 @@ spec:
         fi
 
         if [ "${WORKSPACE_SSH_DIRECTORY_BOUND}" = "true" ] ; then
-          cp -R "${WORKSPACE_SSH_DIRECTORY_PATH}" "${PARAM_USER_HOME}"/.ssh
+          mkdir -p "${PARAM_USER_HOME}"/.ssh
           chmod 700 "${PARAM_USER_HOME}"/.ssh
```

Comment:
usually we use `install -m0700 -D` for that to get the two mkdir and chmod as one command.

(but no need to change since I am not even sure install is avail and it's just the same)